themes/digital.gov/layouts/services/single.html

```diff
@@ -1,66 +1,57 @@
 {{- define "content" -}}
+  {{ $path := "" }}{{ with .File }}
+    {{ $path = .Path }}
+  {{ else }}
+    {{ $path = .Path }}
+  {{ end }}
   <main role="main" id="main-content">
-    <article>
+    <article class="grid-container grid-container-desktop">
       <header>
-        <div class="grid-container grid-container-desktop">
-          {{- partial "partials/core/usa-breadcrumbs.html" . -}}
-          <div class="grid-row">
-            <div class="grid-col-12">
-              {{- if .Params.kicker -}}
-                <p class="kicker">{{- .Params.kicker -}}</p>
-              {{- end -}}
-              {{/* Page Title */}}
-              <h1>{{- .Title | markdownify -}}</h1>
-
-              {{/* Deck */}}
-              {{- if .Params.summary -}}
-                <div class="deck">{{- .Params.summary | markdownify -}}</div>
-              {{- end -}}
-
-              {{- if .Params.source_url -}}
-                <p>
-                  <a
-                    href="{{- .Params.source_url -}}"
-                    title="Go to: {{- .Title | markdownify -}}"
-                    >{{- .Params.source_url -}}</a
-                  >
-                </p>
-              {{- end -}}
-            </div>
+        <div class="grid-row" data-edit-this="{{- $path -}}">
+          <div class="grid-col-12">
```

Comment:
The `grid-row` and `grid-col-12` aren't necessary here because they're full width anyways.